pages/packingDetailed.js

```diff
@@ -0,0 +1,333 @@
+                class="form-control w-full text-base font-normal text-gray-700 bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+                id="pantry_note"
+                rows="4"
+                placeholder="Leave a note here for other pantry workers!"
+              >
+                {this.state.pantryNote}
+              </textarea>
+              <div>
+                <React.Fragment>
+                  <button
+                    class="bg-blue-500 hover:bg-blue-700 text-white py-1 px-2 border border-blue-700 rounded"
+                    onClick={() =>
+                      this.savePantryNote(
+                        document.getElementById("pantry_note").value
+                      )
+                    }
+                  >
+                    Save
+                  </button>
+                  <button
+                    class="bg-white-500 hover:bg-white-700 text-blue py-1 px-2 border border-blue-700 rounded"
+                    onClick={() => this.cancelPantryNote()}
+                  >
+                    Cancel
+                  </button>
+                </React.Fragment>
+              </div>
+            </div>
+            <div className="w-3/4 p-5 space-x-10 space-y-5">
+              {this.state.error ? errorBanner : null}
+              {this.state.success ? successBanner : null}
+              <h1 className="inline text-2xl font-medium mb-2">
+                {this.state.firstName + " " + this.state.lastInitial}{" "}
+              </h1>
+              <div className="inline items-center space-x-4">
+                <h2 className="inline text-l font-medium mb-2">
+                  {this.state.delivery_date}{" "}
+                </h2>
+                {this.displayOrderStatus()}
+                {this.displayChangeOrderStatus()}
+              </div>
+              <div className="text-large font-medium">
+                {"Additional Note: " + this.state.guestNote}
+              </div>
+              <table className="w-full table-fixed my-5" id="order">
+                <thead>
+                  <tr className="border-b-2">
+                    <th className="w-auto text-left">Item</th>
+                    <th className="w-1/6 text-left">Quantity</th>
+                    <th className="w-5"></th>
+                  </tr>
+                </thead>
+                <tbody className="divide-y">
+                  {Object.entries(this.state.items).map(([barcode, value]) =>
+                    this.state.items[barcode].isPacked
+                      ? this.displayPackedItemRow(barcode, value)
+                      : this.displayUnpackedItemRow(barcode, value)
+                  )}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </Layout>
+      </>
+    );
+  }
+}
+
+export default function Checkout() {
```

Comment:
Not sure if this name matters or is actually used anywhere, but probably should change it in case